app/channels/conversation_channel.rb

```diff
@@ -0,0 +1,52 @@
+class ConversationChannel < ApplicationCable::Channel
+  def subscribed
```

Comment:
I would have do this: 
Define a method called conversation where you find the conversation
And then instead of `stream_from "Conversation-#{conversation_id}"`, you can just do `stream_for conversation`